rest-api/appengine_config.py

```diff
@@ -9,3 +9,10 @@
 vendor.add('lib-common')
 vendor.add('appengine-mapreduce/python/src')
 vendor.add('appengine-mapreduce/python/test')
+
+
+def webapp_add_wsgi_middleware(app):
```

Comment:
I don't fully understand where this gets called from, but I suspect it will get called in prod, too... this is the thing that's potentially going to slow things down in prod, not the actual handler in app.yaml.

We should either add an if statement here to check the environment (and just return app in prod), or add a build step so this code doesn't get included when deploying to prod.

Reply:
(If this only gets invoked when appstats is set to on, maybe this isn't necessary once you move the appstats: on to app_nonprod.yaml... but I'd want to confirm that's the case.)

Reply:
according to google, appstats looks for this function name, so it shouldnt get called in prod, but I'll work on a safety check to ensure it.